shared/markdown/parser.pegjs

```diff
@@ -1,8 +1,15 @@
 {
 	const linkExp = new RegExp(/(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)\b/, 'gi')
+    // Instead of encoding all the bad cases into a more complicated regexp lets just add some simple code here
+    function goodLink (link) {
+        if (link.includes('..'))
```

Comment:
i'm not trying to make this 100% perfect at this point. its rare you have a .. in a url as a url never needs one of those